### DISCUSSION TITLE
How to recompile trivy

### DISCUSSION THREAD

**Discussion by WeaponA** [aquasecurity/trivy#5980] [Documentation]
### Description

I modified some of the source code for trivy and wanted to rewrite it and replace it with trivy on my Linux system. However, I couldn't find how to compile it in the documentation. Using go build to compile locally worked, but if I directly replaced the files under the scanner-trivy image directory /usr/local/bin, I got an error that the --no-progress command wasn't found. I would like to know how to recompile trivy after making modifications. Is my direct replacement of the files the wrong way? 

### Suggestions

How to compile trivy and how to introduce scanner-trivy image

**Comment by itaysk** [comment]
https://aquasecurity.github.io/trivy/v0.48/community/contribute/pr/#development
